README.md

```diff
@@ -124,7 +128,9 @@ config.nativeCrashReportEnabled = true;
 // Optional: sample RUM sessions (here, 80% of session will be sent to Datadog. Default = 100%)
 config.sessionSamplingRate = 80;
 // Optional: sample tracing integrations for network calls between your app and your backend (here, 80% of calls to your instrumented backend will be linked from the RUM view to the APM view. Default = 20%)
+// You need to specify the hosts of your backends to prevent tracing with third-party backends
```

Comment:
I would say it other way around: you need to specify the hosts that should be traced.

Because it is not about preventing in the first place, it is about supporting.

Reply:
👍